go/.deploio.yaml

```diff
@@ -1,5 +1,8 @@
+# Be aware if you use nctl to create your app, config settings for the app will be set to 
+# defaults and take precedence over the configs here, except env variables. Configs in this 
+# file are only used if an application is created via API with empty configs.
```

Comment:
This has somewhat changed with https://github.com/ninech/nctl/pull/43, as `nctl` won't set the defaults anymore. So the config here should actually override the settings.